sdlib/d/gc/tcache.d

```diff
@@ -14,6 +14,8 @@ import sdc.intrinsics;
 
 enum DefaultEventWait = 65536;
 
+enum ShouldZeroFreeSlabs = true;
```

Comment:
I suspect you'll have to rename this as it is extended to collection, but let's merge it.